src/modules/Dashboard/components/FlowGrid/FlowGrid.hooks.tsx

```diff
@@ -31,17 +31,15 @@ export function useFlowGridMenu({
   const workspaceRoles = workspaces.useRolesData();
   const roles = appletId ? workspaceRoles?.data?.[appletId] : undefined;
 
-  const canEdit =
-    isManagerOrOwner(roles?.[0]) ||
-    roles?.includes(Roles.Editor) ||
-    roles?.includes(Roles.SuperAdmin);
+  const canEdit = checkIfCanEdit(roles);
 
   const canDoTakeNow =
     featureFlags.enableMultiInformantTakeNow &&
     hasParticipants &&
-    (isManagerOrOwner(roles?.[0]) ||
-      roles?.includes(Roles.Coordinator) ||
-      roles?.includes(Roles.SuperAdmin));
+    (isManagerOrOwner(roles?.[0]) || roles?.includes(Roles.SuperAdmin));
```

Comment:
Any particular reason you opted not to abstract this to a similar  "checkRoles" utility fn? I notice this is the same check made in `ActivityGrid.utils.tsx`.